docs/advanced.rst

```diff
@@ -217,49 +218,49 @@ Change Language
 
   kolibri language setdefault <langcode>
 
-+-----------------------+-----------------+ 
-| Language              | <langcode>      |
-+=======================+=================+ 
-| English               | ``en``          |
-+-----------------------+-----------------+
-| Spanish (Spain)       | ``es-es``       | 
-+-----------------------+-----------------+ 
-| Spanish (Mexico)      | ``es-mx``       | 
-+-----------------------+-----------------+ 
-| French                | ``fr``          | 
-+-----------------------+-----------------+
-| Swahili (Tanzania)    | ``sw-tz``       | 
-+-----------------------+-----------------+
-| Arabic                | ``ar``          | 
-+-----------------------+-----------------+
-| Bulgarian             | ``bg``          | 
-+-----------------------+-----------------+
-| Farsi                 | ``fa``          | 
-+-----------------------+-----------------+
-| Hindi (India)         | ``hi-in``       | 
-+-----------------------+-----------------+
-| Urdu (Pakistan)       | ``ur-pk``       | 
-+-----------------------+-----------------+
-| Marathi               | ``mr``          | 
-+-----------------------+-----------------+
-| Chinyanja             | ``nyn``         | 
-+-----------------------+-----------------+
-| Portuguese (Brasil)   | ``pt-br``       | 
-+-----------------------+-----------------+
-| Telugu                | ``te``          | 
-+-----------------------+-----------------+
-| Vietnamese            | ``vi``          | 
-+-----------------------+-----------------+
-| Yoruba                | ``yo``          | 
-+-----------------------+-----------------+
-| Fulfulde Mbororoore   | ``ff-cm``       | 
-+-----------------------+-----------------+
-| Bengali               | ``bn-bd``       | 
-+-----------------------+-----------------+
-| Gujarati              | ``gu-in``       | 
-+-----------------------+-----------------+
-| Burmese               | ``my``          | 
-+-----------------------+-----------------+
++--------------------------+-----------------+ 
+| Language                 | <langcode>      |
++==========================+=================+ 
+| English                  | ``en``          |
++--------------------------+-----------------+
+| Spanish (Spain)          | ``es-es``       | 
++--------------------------+-----------------+ 
+| Spanish (Latin America)  | ``es-419``      | 
```

Comment:
Haven't followed this change -- will it break existing `es-mx` setups, since they may have a non-existing language code default?

Reply:
We never had `es-mx` in Kolibri, so this is not break anything. We had it for a short time on Studio, but it was never maintained properly. Since it's not sustainable to maintain several Spanish country-locales, we introduced `es-la` (or `es-419` that is used for Latin American Spanish) on Kolibri, and will have one generic `es` on Studio.